packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -361,6 +363,15 @@ class SelectableText extends StatefulWidget {
   /// Otherwise [ColorScheme.primary] of [ThemeData.colorScheme] is used.
   final Color? cursorColor;
 
+  /// The color to use when painting the selection.
+  ///
+  /// If this property is null, this widget gets the selection color from the
+  /// inherited [DefaultSelectionStyle] (if any);
+  /// if none, the selection color is derived from the
+  /// [CupertinoThemeData.primaryColor] on Apple platforms
+  /// and [ThemeData.colorScheme.primary] on other platforms with 40% opacity.
```

Comment:
This line is causing a docs test to fail. Try modifying it to the text below.

`and [ColorScheme.primary] of [ThemeData.colorScheme] on other platforms with 40% opacity.`

Reply:
Thank you 👐🏽